proto/solana/core.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+import "solana/get_public_keys.proto";
+import "solana/sign_txn.proto";
+
+import "error.proto";
+
+package solana;
+
+message Query {
+  oneof request {
+    GetPublicKeysRequest get_public_keys = 1;
+    GetPublicKeysRequest get_user_verified_public_key = 2;
+  }
+}
+
+message Result {
+  oneof response {
+    GetPublicKeysResponse get_public_keys = 1;
+    GetPublicKeysResponse get_user_verified_public_key = 2;
```

Comment:
opinion `get_` prefix does not seem relevant in response
```suggestion
    GetPublicKeysResponse public_keys = 1;
    GetPublicKeysResponse user_verified_public_key = 2;
```

Reply:
referred btc for this, keeping it standard as there

Reply:
okay. makes sense. can pick it all together. resolving for now.